### DISCUSSION TITLE
Data and other boring things - A blog by Qiushi Yan

### DISCUSSION THREAD

**Discussion by giscus** [enixam/qiushi.rbind.io#4] [General]
# Data and other boring things - A blog by Qiushi Yan

Welcome to my blog. This is the place I share notes about various data science technologies, mostly centered around R, Python and Shiny.

https://qiushi.rbind.io/

**Comment by poyone** [comment]
你好，我是从统计之都来的，想问下您目前在哪留学？ 可以交流下嘛，914987163这是我的qq

**Reply by qiushiyan** [reply]
加你了

**Comment by Hywohihihi** [comment]
你好，我正在读你的Forecasting: Principles and Practice第三版的注释，但是练习解决方案已经找不到了。可以加个QQ嘛，1433352665
